packages/sitecore-jss-nextjs/src/index.ts

```diff
@@ -38,6 +38,19 @@ export {
   ComponentFields,
   ComponentParams,
   isServer,
+  LayoutPersonalizationService,
+  PersonalizationResult,
+  PersonalizationLoadResult,
+  LayoutFragmentData,
+  LayoutFragmentService,
+  GraphQLLayoutFragmentService,
+  GraphQLLayoutFragmentServiceConfig,
+  PersonalizationDecisionData,
+  RenderingPersonalizationDecision,
+  PersonalizationDecisionsService,
+  RestPersonalizationDecisionsServiceConfig,
+  RestPersonalizationDecisionsService,
```

Comment:
I do not see GraphQLPersonalizationDecisionsService.
Does it make sense to simplify the names?
E.g. PersonalizationDecisionsService, PersonalizationDecisionsServiceConfig

Does it make sense to have only singular nouns in service names?
E.g. PersonalizationDecisionService, PersonalizationDecisionServiceConfig